cluster/mesos/cluster.go

```diff
@@ -82,13 +83,19 @@ func NewCluster(scheduler *scheduler.Scheduler, TLSConfig *tls.Config, master st
 	// Empty string is accepted by the scheduler.
 	user, _ := options.String("mesos.user", "SWARM_MESOS_USER")
 
+	var found bool
+
+	if cluster.role, found = options.String("mesos.role", "SWARM_MESOS_ROLE"); !found {
```

Comment:
What do you think if we put our `var found bool` inlined with the `if` as follow: 
```
if role, found := options.String("mesos.role", "SWARM_MESOS_ROLE"); found {
 cluster.role = role
}
``` 

And make the default `"*"` part of the initialization of Cluster line 78.